tracee-ebpf/tracee/tracee.go

```diff
@@ -736,6 +736,24 @@ func (t *Tracee) populateBPFMaps() error {
 	bpfConfigMap.Update(uint32(configFollowFilter), boolToUInt32(t.config.Filter.Follow))
 	bpfConfigMap.Update(uint32(configDebugNet), boolToUInt32(t.config.Debug))
 
+	// Initialize pid_to_cont_id_map if tracing containers
+	c := Containers{}
+	err := c.Populate()
+	if err != nil {
+		return fmt.Errorf("failed to populate pid_to_cont_id_map")
+	}
+	bpfPidToContIdMap, _ := t.bpfModule.GetMap("pid_to_cont_id_map")
+	for _, cont_id := range c.GetContainers() {
```

Comment:
```suggestion
	for _, contd := range c.GetContainers() {
```
naming convention: https://golang.org/doc/effective_go#mixed-caps

Reply:
Yep!!! thx